schematools/utils.py

```diff
@@ -117,14 +118,22 @@ def schema_fetch_url_file(schema_url_file):
 
 
 @lru_cache(maxsize=500)  # type: ignore
-def toCamelCase(name):
+def toCamelCase(s: str) -> str:
     """
     Unify field/column/dataset name from Space separated/Snake Case/Camel case
     to camelCase.
     """
-    name = " ".join(name.split("_"))
-    words = RE_CAMEL_CASE.sub(r" \1", name).strip().lower().split(" ")
-    return "".join(w.lower() if i == 0 else w.title() for i, w in enumerate(words))
+    out = StringIO()
+    after_space = False
+    for c in s:
```

Comment:
Single letter variables are really hard to read.
`s==value` and `c==character`?

Otherwise looks nice, though wonder how it will perform and if it will catch nasty edge cases we have (e.g. m2 and stuff).

Reply:
I always use single-letter variable names for generic, short-lived stuff. I find value hard to parse :)

m2?

Reply:
" generic, short-lived stuff." does not apply here, as `s` is effectively base value for function and `c` is accessed 4 times. ;)

m2 = https://github.com/Amsterdam/schema-tools/blob/master/tests/test_utils.py#L31
Would be nice to add test to these 2 cases:
```
    assert toCamelCase("numbers_33_in_the_middle_44") == "numbers33inTheMiddle44"
    assert toCamelCase("per_jaar_per_m2") == "perJaarPerM2"
```
note how `33in` appears over there.

Reply:
I've changed s back to name. That was a leftover from an experimental version.

Reply:
PS: `RE_CAMEL_CASE`, `re` and probably some of `typing` imports are not needed anymore.

Reply:
I don't get why it should be numbers33inTheMiddle44. Why not numbers33InTheMiddle44? Surely "in" is a separate word.

Reply:
Sorry for confusing, and you are totally correct. 
But will this `toCamelCase` generate `"numbers33InTheMiddle44"`? :)

Reply:
Solved.